packages/test-generator/integration-test-templates/cypress/e2e/create-form-spec.cy.ts

```diff
@@ -92,7 +92,23 @@ describe('CreateForms', () => {
   });
 
   describe('DataStoreFormCreateAllSupportedFormFields', () => {
+    it('should save to DataStore with blank values in non-required fields', () => {
+      cy.get('#dataStoreFormCreateAllSupportedFormFields').within(() => {
+        getInputByLabel('String').type('MyString');
+
+        cy.contains('Submit').click();
+
+        cy.contains(/MyString/).then((recordElement: JQuery) => {
+          const record = JSON.parse(recordElement.text());
+
+          expect(record.string).to.equal('MyString');
+        });
+      });
+    });
+
     it('should save to DataStore', () => {
+      // reset state
+      cy.reload();
```

Comment:
This implies to me that as it currently stands, these tests must execute in a specific order to pass. Is it possible to reset state between tests as part of a `beforeEach()`?

Reply:
Good call, will do